### DISCUSSION TITLE
[FR] support templates

### DISCUSSION THREAD

**Discussion by huyz** [coddingtonbear/obsidian-web#58] [Q&A] [answered]
I also use the [`MarkDownload` extension](https://github.com/deathau/markdownload/) which allows me to specify a template like:

```markdown
---
title: '{pageTitle}'
created: '{date:YYYY-MM-DD HH:mmZ}'
tags: ['-clip', '{keywords:', '}']
url: {baseURI}
author: {byline}
---


> [!QUOTE] 🗣 [{pageTitle}]({baseURI})
> > [!META]- Ⓜ️ clipped:: '{date:YYYY-MM-DD HH:mmZ}'
> > author:: {byline}
>
> > [!SUMMARY] 📋 {excerpt}
>
```

Would be nice to have that supported here as well.

**Comment by coddingtonbear** [comment, marked as answer]
You have this capability already.  See the "templates" section from the plugin options page.

**Reply by ghost** [reply]
So is there a way to add author name, publish date etc. like markdownload? I've checked options and tried {{byline}} and things like that, but couldn't figure it out.
